packages/snaps-sdk/CHANGELOG.md

```diff
@@ -6,6 +6,27 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.0]
+### Added
+- **BREAKING:** Add non-restricted RPC method `snap_resolveInterface`, which allows a snap to resolve a given user interface bound to a `snap_dialog` with a custom value. ([#2509](https://github.com/metamask/snaps/pull/2509))
+  - Add new types `ResolveInterfaceParams`, `ResolveInterfaceResult`.
+- feat: Card component ([#2480](https://github.com/metamask/snaps/pull/2480))
+- Add `BoxChildStruct`, `FormChildStruct`, `FieldChildUnionStruct`. ([#2409](https://github.com/metamask/snaps/pull/2409))
+- Add `Container` and `Footer` components ([#2517](https://github.com/metamask/snaps/pull/2517))
+
+### Changed
+- `RootJSXElementStruct` is updated so that the allowed JSX elements at the root are the null union of the Container component and the allowed children of the Box component. ([#2526](https://github.com/metamask/snaps/pull/2526))
+- Bump `@metamask/key-tree` from `^9.1.1` to `^9.1.2` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/rpc-errors` from `^6.2.1` to `^6.3.1` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+- Bump `@metamask/utils` from `^8.3.0` to `^9.1.0` ([#2445](https://github.com/MetaMask/snaps/pull/2445))
+
+### Fixed
+- Add `typedUnion` utility which strictly validates members of the union based on a type field. ([#2534](https://github.com/metamask/snaps/pull/2534))
```

Comment:
This is an internal function, so it doesn't need to be mentioned in the changelog.

```suggestion
```